src/xmlsec/crypto.py

```diff
@@ -1,11 +1,10 @@
 import io
 import os
 import base64
-import logging
 import threading
 import six
 from six.moves import xrange
-from xmlsec import constants
+from xmlsec import constants, log
```

Comment:
Do not do this. Instead grab a new logger instance from `logging`

```python
...
import logging


log = logging.getLogger(__name__)
```

This means that we can configure `xmlsec` differently from `xmlsec.crypto` (these are the loggers' names).

Reply:
Ok, I might have misunderstood Leif, I agree that this is better.